testing/internal/e2e/tests/base/bytes_up_down_test.go

```diff
@@ -143,7 +143,7 @@ func TestCliBytesUpDownTransferData(t *testing.T) {
 			newBytesUp = int(newSessionReadResult.Item.Connections[0].BytesUp)
 			newBytesDown = int(newSessionReadResult.Item.Connections[0].BytesDown)
 
-			if !(newBytesDown > bytesDown) || !(newBytesUp > bytesUp) {
+			if !(newBytesDown > bytesDown) {
```

Comment:
I think we only care about bytesDown (the data transmitted from the session?) in this test case?